docker/vttestserver/Dockerfile.mysql80

```diff
@@ -17,7 +17,7 @@
 #       ensure images contain the right binaries.
 
 # Use a temporary layer for the build stage.
-ARG bootstrap_version=30
+ARG bootstrap_version=31
```

Comment:
Curious why we aren’t using minor versions on main? I feel like we might have discussed this before and I forgot 🙂

Reply:
Technically nothing prevents us from doing this: increasing the major version of bootstrap on main by one every time we cut a new RC-1, and keep increasing its minor version on main.

A while ago we decided to do what we do right now (increment the major on main all the time), but if we reach a consensus, we could change the decision.